database/factories/TaskFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Lightit\Backoffice\Task\Domain\Models\Task;
+use Lightit\Backoffice\Employee\Domain\Models\Employee;
+use Database\Factories\EmployeeFactory;
+
+/**
+* @extends Factory<Task>
+*/
+
+
+class TaskFactory extends Factory
+{
+    /**
+     * @var class-string<\Lightit\Backoffice\Task\Domain\Models\Task>
+     */
+    protected $model = Task::class;
+    public function definition(): array
+    {
+        return [
+            'title' => fake()->title(),
+            'description' => fake()->text(),
+            'status' => 'pending',
+            'employee_id' => EmployeeFactory::new()->create()
```

Comment:
If the `employee_id` is the Employee's primary key, which I understand it is, you can take the `create()` out and Laravel will resolve, by its own, that

```suggestion
            'employee_id' => EmployeeFactory::new()
```